scripts/jenkins-yoctobuild-build.sh

```diff
@@ -141,15 +141,18 @@ prepare_build_config() {
     local client_version=$($WORKSPACE/integration/extra/release_tool.py --version-of mender --in-integration-version HEAD)
     local mender_artifact_version=$($WORKSPACE/integration/extra/release_tool.py --version-of mender-artifact --in-integration-version HEAD)
 
+
+    local mender_binary_delta_version=$($WORKSPACE/mender-binary-delta/x86_64/mender-binary-delta --version | egrep -o '[0-9]+\.[0-9]+\.[0-9b]+(-build[0-9]+)?')
```

Comment:
Looks good.

I usually separate the beta part explicitly, like `[0-9]+\.[0-9]+\.[0-9]+(b[0-9]+)?(-build[0-9]+)?`. But it is just personal taste.

Reply:
Yes, strictly speaking that is better. But given the length of these release pipelines, I think I will fix that in a followup PR, not here. :-)

Reply:
Agreed, do not launch a new pipeline just for this. There is no problem with leaving it as-is neither, but just a note for future reference (we use these kind of regex in many places)